assets/quick-order-list.js

```diff
@@ -1,467 +1,510 @@
-class QuickOrderListRemoveButton extends HTMLElement {
-  constructor() {
-    super();
-    this.addEventListener('click', (event) => {
-      event.preventDefault();
-      const quickOrderList = this.closest('quick-order-list');
-      quickOrderList.updateQuantity(this.dataset.index, 0);
-    });
-  }
-}
-
-customElements.define('quick-order-list-remove-button', QuickOrderListRemoveButton);
-
-class QuickOrderListRemoveAllButton extends HTMLElement {
-  constructor() {
-    super();
-    const allVariants = Array.from(document.querySelectorAll('[data-quantity-variant-id]'));
-    const items = {}
-    let hasVariantsInCart = false;
-    this.quickOrderList = this.closest('quick-order-list');
-    allVariants.forEach((variant) => {
-      const cartQty = parseInt(variant.dataset.cartQuantity);
-      if (cartQty > 0) {
-        hasVariantsInCart = true;
-        items[parseInt(variant.dataset.quantityVariantId)] = 0;
-      }
-    });
-
-    if (!hasVariantsInCart) {
-      this.classList.add('hidden');
-    }
-
-    this.actions = {
-      confirm: 'confirm',
-      remove: 'remove',
-      cancel: 'cancel'
+if (!customElements.get('quick-order-list-remove-button')) {
```

Comment:
We actually already have these changes on `main` . It was from the quick add bulk PR

https://github.com/Shopify/dawn/blob/main/assets/quick-order-list.js#L1

I recommend rebasing off of main

Reply:
oh amazing 🙏 🙏 🙏 🙏